content/en/docs/tutorials/cluster-management/install-use-dra.md

```diff
@@ -0,0 +1,540 @@
+    ```
+
+    Looking deeper at the `some-gpu` ResourceClaim, you can see that the status stanza includes information about the
+    device that has been allocated and for what pod it has been reserved for:
+
+    ```shell
+    kubectl get resourceclaim some-gpu -n dra-tutorial -o yaml
+    ```
+
+    The output is similar to this:
+    {{< highlight yaml "linenos=inline, hl_lines=30-33 41-44, style=emacs" >}}
+    apiVersion: v1
+    items:
+    - apiVersion: resource.k8s.io/v1beta2
+      kind: ResourceClaim
+      metadata:
+        creationTimestamp: "2025-07-29T05:11:52Z"
+        finalizers:
+        - resource.kubernetes.io/delete-protection
+        name: some-gpu
+        namespace: dra-tutorial
+        resourceVersion: "58357"
+        uid: 79e1e8d8-7e53-4362-aad1-eca97678339e
+      spec:
+        devices:
+          requests:
+          - exactly:
+              allocationMode: ExactCount
+              count: 1
+              deviceClassName: gpu.example.com
+              selectors:
+              - cel:
+                  expression: device.capacity['gpu.example.com'].memory.compareTo(quantity('10Gi'))
+                    >= 0
+            name: some-gpu
+      status:
+        allocation:
+          devices:
+            results:
+            - adminAccess: null
+              device: gpu-4
+              driver: gpu.example.com
+              pool: kind-worker
+              request: some-gpu
+          nodeSelector:
+            nodeSelectorTerms:
+            - matchFields:
+              - key: metadata.name
+                operator: In
+                values:
+                - kind-worker
+        reservedFor:
+        - name: pod0
+          resource: pods
+          uid: fa55b59b-d28d-4f7d-9e5b-ef4c8476dff5
+    kind: List
+    metadata:
+      resourceVersion: ""
+    {{< /highlight >}}
+
+1.  Observe the driver by checking the pod logs for pods backing the driver
+   daemonset:
+
+    ```shell
+    kubectl logs -l app.kubernetes.io/name=dra-example-driver -n dra-tutorial
+    ```
+
+    The output is similar to this:
+    ```
+    I0729 05:11:52.679057       1 driver.go:84] NodePrepareResource is called: number of claims: 1
+    I0729 05:11:52.684450       1 driver.go:112] Returning newly prepared devices for claim '79e1e8d8-7e53-4362-aad1-eca97678339e': [&Device{RequestNames:[some-gpu],PoolName:kind-worker,DeviceName:gpu-4,CDIDeviceIDs:[k8s.gpu.example.com/gpu=common k8s.gpu.example.com/gpu=79e1e8d8-7e53-4362-aad1-eca97678339e-gpu-4],}]
+    ```
+
+You have now successfully deployed a Pod with a DRA based claim, and seen it
+scheduled to an appropriate node and the associated DRA APIs updated to reflect
+its status.
+
+## Remove the Pod with a claim
+
+When a Pod with a claim is deleted, the DRA driver deallocates the resource so
+it can be available for future scheduling. You can observe that by deleting our
+pod with a claim and seeing that the state of the ResourceClaim changes.
```

Comment:
```suggestion
it can be available for future scheduling. To validate this behavior, delete the Pod that you
created in the previous steps and watch the corresponding changes to the ResourceClaim
and driver.
```

Reply:
Fixed in a follow up PR